deployment/k8s/texera-helmchart/files/texera_ddl.sql

```diff
@@ -101,6 +101,7 @@ CREATE TABLE IF NOT EXISTS "user"
     google_id     VARCHAR(256) UNIQUE,
     google_avatar VARCHAR(100),
     role          user_role_enum NOT NULL DEFAULT 'INACTIVE',
+    comment       TEXT,
```

Comment:
This change also seems unintentional.

Reply:
This is needed, because the latest master add this column to the DB